src/Ports/FileHelper.php

```diff
@@ -0,0 +1,41 @@
+<?php
+declare(strict_types=1);
+
+namespace Shouze\ParkedLife\Ports;
+
+class FileHelper
+{
+    public function readIterator(string $filename): \Iterator
+    {
+        try {
+            $fileObject = new SplFileObject($filename);
+            $fileObject->setFlags(SplFileObject::SKIP_EMPTY | SplFileObject::READ_AHEAD | SplFileObject::DROP_NEW_LINE);
+        } catch (\Exception $e) {
+            throw new \LogicException(sprintf('Unable to build a read iterator for file "%s"', $filename), 0, $e);
+        }
+
+        return $fileObject;
+    }
+
+    public function appendSecurely(string $filename, string $content)
```

Comment:
Still no fan of that method name, as it's a helper I think we don't have to care to much ATM.